dockerfiles/verify_packages/installer/test_install_tracer_appsec_latest.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env sh
+
+set -e
+
+. "$(dirname ${0})/utils.sh"
+
+# Initially no ddtrace/ddappsec
+assert_no_ddtrace
+assert_no_ddappsec
+
+# Install latest appsec and ddtrace using the php installer
+php dd-library-php-setup.php --php-bin php
+
+assert_ddtrace_installed
+assert_ddappsec_installed
+
+assert_file_exists "$(php_conf_dir)"/98-ddtrace.ini
+assert_file_exists "$(php_conf_dir)"/98-ddappsec.ini
```

Comment:
This decision is still pending. @cataphract are you ok with using one file only?

Reply:
It wouldn't be my preference, but I can live with it.